### PR TITLE
Move Application:: hide_application, hide_others and set_menu to Mac platform extension

### PR DESCRIPTION
**Pull request by Maan2003**

### CODE REVIEW

File: druid/src/command.rs [hunk 1]
```diff
@@ -183,10 +183,18 @@ pub mod sys {
     /// Quit the running application. This command is handled by the druid library.
     pub const QUIT_APP: Selector = Selector::new("druid-builtin.quit-app");
 
-    /// Hide the application. (mac only?)
+    /// Hide the application. (mac only)
+    #[cfg_attr(
```
Comment (cmyr) [inline druid/src/command.rs:187]: I spent a few minutes thinking about whether this is too fancy, but I think it's a good change. 👍

**Review by cmyr — APPROVED** [review]
Love to see this, thanks 🥳

**Comment by Maan2003** [comment]
why are `menu::sys::{os}` not behind cfg?

**Comment by cmyr** [comment]
> why are `menu::sys::{os}` not behind cfg?

not for any concrete reason.

**Comment by Maan2003** [comment]
lets leave it for now and allow(deprecated). I would prefer it to be separate PR, if needed

**Pull request merged**